network-api/networkapi/buyersguide/templates/buyersguide/catalog.html

```diff
@@ -114,10 +114,10 @@ <h1 class="h1-heading text-center mt-0">{{ page.header }}</h1>
               <source
                 media="(max-width: 414px)"
```

Comment:
note that as we're reducing the source to a universal "same x1 and x2 at all sizes", we don't need this `media` attribute anymore, and the other `<source>` collections still need to be removed